json.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: ['plugin:json/recommended-with-comments'],
```

Comment:
Note from Jordi:
Adds all json rules (https://github.com/azeemba/eslint-plugin-json) It relies on VSCodes' implementation for validating JSON validation. Allows comments so that json files that do allow comments don't have get marked with an error. This is useful for if someone updates a json file on github.com, so that it fails the build if there was an error. Before we had the json plugin put in so that json files could be linted, but it didn't actually give any errors because all the rules were off.